pxtarget.json

```diff
@@ -465,7 +465,8 @@
         "chooseLanguageRestrictionOnNewProject": true,
         "openProjectNewTab": true,
         "hasReferenceDocs": true,
-        "python": true
+        "python": true,
+        "embedBlocksInSnapshot": true
```

Comment:
this breaks transparency of screenshots right?

Reply:
yeah :( converting between html canvas <-> png erases any rgb data encoded in transparent pixels and i haven't figured out a good workaround... i think i tried giving them all 0.1 opacity or something at the time and it didn't work either but i didn't have time to investigate why